src/com/nicoles_change_maker/CoinMap.java

```diff
@@ -0,0 +1,23 @@
+package com.nicoles_change_maker;
+import java.util.Iterator;
+import java.util.LinkedHashMap;
+/**
+ * Created by bears8yourface on 1/5/16.
+ */
+public class CoinMap {
+
+    public static final LinkedHashMap dictionary = new LinkedHashMap<String, Double>() {
+        {
+            put("H", .50);
+            put("Q", .25);
+            put("D", .10);
+            put("N", .05);
+            put("P", .01);
+        }
+    };
+
+    public Iterator getIterator() {
```

Comment:
In typed languages, encoding types in names is a smell: it duplicates information that's already encoded in the type! I know that this method returns an `Iterator`, so naming it `getIterator` doesn't add much. What is it actually doing? (That is, what does this thing iterate over?)